scripts/postlink/ios/postlink.js

```diff
@@ -129,11 +156,13 @@ function getPlistPath(){
     }
 
     var INFO_PLIST_PROJECT_KEY = 'INFOPLIST_FILE';
-    var plistPathValue = parsedXCodeProj.getBuildProperty(INFO_PLIST_PROJECT_KEY);
+    var targetProductName = package ? package.name : null;
+
+    var plistPathValue = getBuildSettingsPropertyMatchingTargetProductName(parsedXCodeProj, INFO_PLIST_PROJECT_KEY, targetProductName);
```

Comment:
Is it okay to assume that we want the 'Release' build? And if we do put 'Release' here, will the function you added successfully pick the Release one?